examples/quickstart/wasmcloud-host.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: k8s.wasmcloud.dev/v1alpha1
+kind: WasmCloudHostConfig
+metadata:
+  name: wasmcloud-host
+spec:
+  lattice: default
+  version: "1.0.4"
+  natsAddress: nats://nats-cluster.default.svc.cluster.local
```

Updated for the deployment instructions change:
```suggestion
  natsAddress: nats://nats.default.svc.cluster.local
```